Dockerfile

```diff
@@ -3,18 +3,57 @@ FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y --no-install-recommends \
   build-essential \
-  cmake \  
-  qtbase5-dev \
-  libqt5svg5-dev \
-  libqt5websockets5-dev \
-  libqt5opengl5-dev \
-  libqt5x11extras5-dev \
+  cmake \
   libbfd-dev \
   libdwarf-dev \
   libdw-dev \
-  libbz2-dev \
-  libcapnp-dev \
+  # libbz2-dev \
+  # libcapnp-dev \
   python3 \
-  python3-pip
+  python3-pip \
+  wget
+
+RUN cd /tmp && \
+    VERSION=0.7.0 && \
+    wget --no-check-certificate https://capnproto.org/capnproto-c++-${VERSION}.tar.gz && \
+    tar xvf capnproto-c++-${VERSION}.tar.gz && \
+    cd capnproto-c++-${VERSION} && \
+    CXXFLAGS="-fPIC" ./configure && \
+    make -j$(nproc) && \
+    make install
+
+RUN cd /tmp && \
+    wget --no-check-certificate ftp://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz && \
+    tar xvfz bzip2-1.0.8.tar.gz && \
+    cd bzip2-1.0.8 && \
+    CFLAGS="-fPIC" make -f Makefile-libbz2_so && \
+    make && \
+    make install
+
+RUN apt-get update
+
 RUN pip3 install jinja2
-ENV PYTHONPATH /tmp/plotjuggler/3rdparty
\ No newline at end of file
+ENV PYTHONPATH /tmp/plotjuggler/3rdparty
+
+RUN echo "deb http://archive.ubuntu.com/ubuntu bionic main restricted" > /etc/apt/sources.list.d/bionic.list
+RUN echo "deb http://archive.ubuntu.com/ubuntu bionic-updates main restricted" >> /etc/apt/sources.list.d/bionic.list
+
+RUN echo "deb http://archive.ubuntu.com/ubuntu bionic universe" >> /etc/apt/sources.list.d/bionic.list
```

Comment:
figure out which repo is necessary and just and add it with add-apt-repository  before the apt-get update